src/ServiceControl.Monitoring.Tests/Infrastructure/VariableHistoryIntervalStoreTests.cs

```diff
@@ -39,9 +39,28 @@ public void Store_updates_all_supported_historical_periods()
                 Assert.AreEqual(1L, intervals[0].TotalMeasurements);
             }
         }
-    }
 
-    public class SomeStore : VariableHistoryIntervalStore
-    {
+        class BreakdownId
```

Comment:
There's no generic requirement for the type of breakdown. Why not to use some primitive, like `int`?